configuration/bahmniforms/obstetrical_examination_1.json

```diff
@@ -7,57 +7,83 @@
         "published": true,
         "auditInfo": {
             "creator": {
-                "uuid": "0e13890d-da59-11ee-85e8-0242ac140008",
-                "display": "admin",
+                "uuid": "A4F30A1B-5EB9-11DF-A648-37A07F9C90FB",
+                "display": "daemon",
```

Comment:
Let's leave such fields untouched. You only want to copy over the `"resources":` section with the necessary translations, not forgetting to update the relevant reference UUIDs.

Reply:
This has been handled [here](https://github.com/CRUDEM/openmrs-config-hsc/pull/74/commits/2712245a8e29bdc4c82a10a103954ed4549fa154)

Reply:
[Screencast from 07-05-24 12:08:05.webm](https://github.com/CRUDEM/openmrs-config-hsc/assets/36139599/59b86f41-4ba5-415a-8ad2-f4da6a207492)